### PR TITLE
Added context link

### PR DESCRIPTION
**Pull request by nosklo**
So that people arriving here can know what this is about

### CODE REVIEW

**Review by Zayanusman221 — COMMENTED** [review]
.

**Comment by Zayanusman221** [comment]
> .

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Signal TLS Proxy
```
Comment (KashyGodson001) [inline README.md:1]: https://github.com/signalapp/Signal-TLS-Proxy/pull/27/files

**Comment by jon-signal** [comment]
Hello, and thank you for the contribution! While I think this makes perfect sense, I'm going to close this PR (with our gratitude!) because:

1. We'd need you to [sign the Contributor License Agreement (CLA)](https://signal.org/cla/) and
2. We're planning on giving the README a bit of a refresh shortly anyhow

I'm sorry it took so long to get to this, but thank you again for your effort!

**Pull request closed without merging**